### PR TITLE
Lab 07

### PR DESCRIPTION
**Pull request by bodnarira**

### CODE REVIEW

File: Lab07.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import urllib.request
+
+class File:
+  def __init__ (self, url):
+    self.data = urllib.request.urlopen(url).read().decode("utf-8")
```
Comment (polyakovartem) [inline Lab07.py:6]: if the file is 10GB?

**Review by polyakovartem — CHANGES_REQUESTED** [review]
word statistics is missing

File: Lab07.py (hunk 1 above)
Comment (bodnarira) [reply Lab07.py:6]: I cant find so large text file. Could you send me a link?

Comment (polyakovartem) [reply Lab07.py:6]: you don't need 10GB file here to understand what will happen. On the other hand if you need it, you can generate one

**Pull request closed without merging**